cmd/krew/cmd/search.go

```diff
@@ -118,6 +121,53 @@ Examples:
 	PreRunE: checkIndex,
 }
 
+func searchByNameAndDesc(keyword string, target searchTarget) []string {
+	if keyword == "" {
+		return target.names
+	}
+
+	var searchResults = make([]string, 0, len(target.names))
+
+	// find by names
+	matches := fuzzy.Find(keyword, target.names)
+	searchResultsMap := make(map[int]bool)
+	for _, m := range matches {
+		searchResults = append(searchResults, m.Str)
+		searchResultsMap[m.Index] = true
+	}
+
+	// find by short descriptions
+	fuzzyMatches := fuzzy.Find(keyword, target.descriptions)
```

Comment:
nit: `matches` above are technically `fuzzyMatches`, too. maybe reuse that variable?

Reply:
Updated.
PTAL thanks.